packages/core/src/tracing/clsHooked/emitter-listener.js

```diff
@@ -4,6 +4,17 @@
  *
  * Original license: BSD 2 Clause
  * (see https://github.com/othiym23/emitter-listener/blob/7586fba839cf87774d5df224ce479c3b7e2b9964/package.json#L26)
+ *
+ * A note on the usage of the properties __wrappedIn, __unwrapIn, __wrapped  and __unwrap: The original package
+ * (https://github.com/othiym23/emitter-listener/blob/af156cef522f1daa11523b2d94b6f7fd1e859ca5/listener.js) attaches
+ * properties named __wrapped and __unwrap to the event emitter object. Even with this this fixed and vendored
```

Comment:
`this` is written twice in a row. Not sure if it was intentional as the sentence would more or less still make sense with or without them.

Reply:
That was a typo, thanks. Fixed now.